pkg/client/clientset_generated/release_1_4/import_known_versions.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package release_1_4
+
+// These imports are the API groups the client will support.
+import (
+	"fmt"
+
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
@caesarxuchao Done.

Since it was a tiny change, I also squashed the commit already.